client/app/pages/queries-list/index.js

```diff
@@ -1,12 +1,23 @@
 import moment from 'moment';
+import { isString } from 'underscore';
+import startsWith from 'underscore.string/startsWith';
```

Comment:
You can use `String`'s own `startsWith`.

Reply:
Hm, I thought it's not supported by IE11?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith#Browser_compatibility

Reply:
Thinking about IE11 is very benevolent of you :) But I would assume Babel takes care of this for us?